untie/src/untie.h

```diff
@@ -0,0 +1,20 @@
+
+#pragma once
+
+#include "knoting/engine.h"
+
+class Window;
```

Comment:
Shouldn't this forward declaration be inside the `knot` namespace?

Reply:
it should c: